packages/shared-ux/prompt/errors/src/not_found.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect, useState, useCallback } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiEmptyPrompt,
+  EuiEmptyPromptProps,
+  EuiImage,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+const NOT_FOUND_TITLE = i18n.translate('sharedUXPackages.prompt.errors.notFound.title', {
+  defaultMessage: 'Page not found',
+});
+
+const NOT_FOUND_BODY = i18n.translate('sharedUXPackages.prompt.errors.notFound.body', {
+  defaultMessage:
+    "Sorry, the page you're looking for can't be found. It might have been removed or renamed, or maybe it never existed at all.",
+});
+
+const NOT_FOUND_GO_BACK = i18n.translate('sharedUXPackages.prompt.errors.notFound.goBacklabel', {
+  defaultMessage: 'Go back',
+});
+
+interface NotFoundProps {
```

Comment:
nit, unless this is for documentation purposes.

```suggestion
export type NotFoundProps = Pick<EuiEmptyPromptProps, 'actions'>;
```

Reply:
Indeed I wrote it as an interface to have inline documentation for the prop.

<img width="585" alt="Screenshot 2022-11-29 at 10 45 02" src="https://user-images.githubusercontent.com/57448/204495148-2107cc5f-8010-4845-9bd9-518a94fa266d.png">